### DISCUSSION TITLE
Two CRD of helmreleases

### DISCUSSION THREAD

**Discussion by sveinpj** [fluxcd/flux2#3975] [Q&A] [answered]
In my 'kubectl get crd | grep helmreleases', I got two helmreleases
helmreleases.helm.fluxcd.io
helmreleases.helm.toolkit.fluxcd.io

If I type 'kubectl get hr', I got "No resources found"
If I type 'k get helmreleases.helm.toolkit.fluxcd.io', I got the installed HR 

How do I get rid of helmreleases.helm.fluxcd.io?
I tried to remove the (legacy) CRD, but 'kubectl get hr' came back with:
Error from server (NotFound): Unable to list "helm.fluxcd.io/v1, Resource=helmreleases": the server could not find the requested resource (get helmreleases.helm.fluxcd.io)

**Comment by stefanprodan** [comment, marked as answer]
After deleting the old CRD, you also need to clear the kubectl local cache by deleting the contents of `.kube/cache`

**Reply by sveinpj** [reply]
That was too easy..hehe. Thanx mate :-)
